src/tim/layout_infer/layout_inference.cc

```diff
@@ -84,7 +84,8 @@ bool LayoutInferContext::IsVisited(const std::shared_ptr<vx::Operation>& op) con
 bool LayoutInferContext::IsReadyForInfer(
     const std::shared_ptr<vx::Operation>& op) const {
   for (const auto& tensor : op->impl()->InputsTensor()) {
-    if (tensor_pv_.end() == tensor_pv_.find(tensor)) {
+    if (tensor_pv_.end() == tensor_pv_.find(tensor) &&
+        !tensor->IsConstTensor()) {
```

Comment:
!tensor->IsConstTensor() && (tensor_pv_.end() == tensor_pv_.find(tensor))
reorder the condition for performance reason.